metabrainz/templates/users/account-type.html

```diff
@@ -6,25 +6,24 @@
   <h1 class="page-title">{{ _('Support us') }}</h1>
 
   <p>
-    {{ _('For commercial use of our data or to get access to the
+    {{ _('If you rely on data from MetaBrainz, please remember that *we* rely on contributions 
```

Comment:
I think the emphasis on *we* is not quite apt. First, it should bold or italic, not in * *, since * * is a way to make things bold in plain text, but we're on a web page. Second, I think that emphasis is a bit unprofessional -- I would just nuke it.